### PR TITLE
POC/adaptive cards

### PR DESCRIPTION
**Pull request by Sanjuice**
This is a proof of concept for adaptive cards using javascript.

### CODE REVIEW

File: examples/adaptive-cards/rollup.config.js
Comment (kruplm) [inline examples/adaptive-cards/rollup.config.js:1]: Please add a license header here too, like the one in the index.html

File: examples/adaptive-cards/app.js
Comment (kruplm) [inline examples/adaptive-cards/app.js:1]: Please add a license header here too.

File: examples/adaptive-cards/README.md
Comment (kruplm) [inline examples/adaptive-cards/README.md:1]: Please add a license header here too.

**Review by kruplm — CHANGES_REQUESTED** [review]

File: package.json [hunk 1]
```diff
@@ -3,10 +3,11 @@
   "description": "Lerna monorepo of the web dependencies in ComposeUI",
   "private": true,
   "workspaces": [
+    "examples/adaptive-cards",
     "examples/js-chart-and-grid-messagerouter/js-*",
     "examples/fdc3-chart-and-grid/js-*",
     "examples/fdc3-pricing-and-chat/js-*",
-	"examples/fdc3-trade-simulator/js-*",
+	  "examples/fdc3-trade-simulator/js-*",
```
Comment (lilla28) [inline package.json:10]: can you please fix the identation here fully?

File: examples/adaptive-cards/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+## During Development
+
+```npm i
+
+npx lerna run start --stream --scope=@morgan-stanley/composeui-example-adaptive-cards
+```
+
+<!---
+composeui .\Tryouts\Plugins\ApplicationPlugins\defaultApp\app.js --->
```
Comment (lilla28) [inline examples/adaptive-cards/README.md:16]: What is this comment here?

**Comment by lilla28** [comment]
Can you please also squash your commits as well? as they are mostly belongs together (creating adaptive card poc for JS)

File: examples/adaptive-cards/README.md (hunk 2 above)
Comment (Sanjuice) [reply examples/adaptive-cards/README.md:16]: It was part of the default app. It's been removed.

File: package.json (hunk 1 above)
Comment (Sanjuice) [reply package.json]: Fixed it :)

**Review by kruplm — APPROVED** [review]
LGTM

**Review by lilla28 — APPROVED** [review]

**Review by fhubi — APPROVED** [review]

**Pull request merged**